plugins/woocommerce/src/Admin/API/OnboardingFreeExtensions.php

```diff
@@ -68,12 +73,29 @@ public function get_items_permissions_check( $request ) {
 	/**
 	 * Return available payment methods.
 	 *
-	 * @param \WP_REST_Request $request Request data.
+	 * @param WP_REST_Request $request Request data.
 	 *
-	 * @return \WP_Error|\WP_REST_Response
+	 * @return WP_Error|WP_REST_Response
 	 */
 	public function get_available_extensions( $request ) {
-		return RemoteFreeExtensions::get_extensions();
+		$extensions = RemoteFreeExtensions::get_extensions();
+		/**
+		* Remove Jetpack, WooCommerce Shipping & Tax when woocommerce_suggest_jetpack is false.
+		 *
+		 * @since 7.8
+		*/
+		if ( false === apply_filters( 'woocommerce_suggest_jetpack', true ) ) {
```

Comment:
I'm wondering if `"woocommerce_onboarding_suggest_jetpack"` would be more explicit. What do you think?

Reply:
That was my first thought as well. However, I think we plan to use `woocommerce_suggest_jetpack` on other pages where we also suggest Jetpack.

Reply:
Yup we should roll this out everywhere Jetpack is suggested 👍 

 I'll audit wc-admin for this and create a new issue.